core/timers/timers.ts

```diff
@@ -60,9 +60,7 @@ export class Timers {
       // Mooneye Test, tima_write_reloading
       // Writing in this strange delay cycle, will cancel
       // Both the interrupt and the TMA reload
-      if (Timers.timerCounterOverflowDelay) {
-        Timers.timerCounterOverflowDelay = false;
-      }
+      Timers.timerCounterOverflowDelay = false;
```

Comment:
So this may work, but it may fail some of the accuracy tests, if you put some things back, and they are still failing, check this 😄